src/SFA.DAS.EmployerIncentives.TestHelpers/Images/Dockerfile

```diff
@@ -0,0 +1,35 @@
+﻿FROM mcr.microsoft.com/mssql/server:2019-latest
+ENV ACCEPT_EULA=Y
+ENV SA_PASSWORD=Pa55word!
```

Comment:
Password in plain text?

Reply:
This is just for the test db.  Could try and replace it with the test Instance Id I suppose.  Easier for the devs if it's the same value though.